firebase-functions/gradle.properties

```diff
@@ -1,3 +1,3 @@
-version=16.2.0
-latestReleasedVersion=16.1.3
+version=16.3.1
+latestReleasedVersion=16.3.0
```

Comment:
this seems odd, can you verify it is intended?

Reply:
confirmed